--- 
title: "Predicting pollination phenology in lodgepole pine"
author: "C. Susannah Tysor"
date: "`r Sys.Date()`"
site: bookdown::bookdown_site
output: bookdown::pdf_book
documentclass: article
bibliography: [literature.bib, packages.bib]
biblio-style: apalike
link-citations: yes
fontsize: 12pt
description: "This paper describes a model of pollination phenology"
---
# Figure captions

(ref:rangemap) Rangemap of lodgepole pine. After @Little1971

(ref:sitemap) Map of seed orchard locations. Seed orchard locations are in red. Weather data gridpoints are in blue. Boxed area in map on left is shown in greater detail on the right.

(ref:spu) Seed planning units are biogeoclimatic and political units used for seed planting purposes by British Columbia. Seed planning units form this project's provenances. High, Low, and Mid refer to elevational bands.

(ref:ppccum) Cumulative distribution of accumulated forcing for observed phenophases and predicted states

(ref:ppcprop) Proportion of states correctly predicted by the model for each phenophase.

(ref:ppcsite) Proportion of states correctly predicted by the model for each phenophase and site.

(ref:ppccounts) Count of states predicted at each observed phenophase.

(ref:fstartfend) $fstart$ and $fend$ model estimates

(ref:apcforce) Absolute value of the change in $fstart$ with standard error

(ref:apcday) Absolute value of the change in start day for 30 years of accumulated forcing time series with standard error. Accumulated forcing time series are ordered from in terms of forcing accumulated at day 151, the mean day of year for phase 2 in the phenology data.

(ref:surveys) Days orchards were surveyed for phenology in each year.

(ref:floweringperiod) Flowering period date predictions from the model in 15 years at 3 sites. Receptivity in purple, pollen shed in green. Gray bars show max and min extent of flowering period in data and black bars show max and min flowering for years and sites with data.

(ref:floweringlength) Length of flowering period at 3 sites.

(ref:overlap) Receptivity overlap with pollen shed at 3 sites under conditions in 1997-2011, and with 2 and 5 degrees of warming.

(ref:siteprovclimate) Provenance (SPU) climates of data included in this proposal, provenances not included in this proposal, and sites where provenances were grown. MAT is mean annual temperature and MAP is mean annual precipitation.


```{r include=FALSE}
# automatically create a bib database for R packages
knitr::write_bib(c(
  .packages(), 'bookdown', 'knitr', 'rmarkdown'
), 'packages.bib')
```


```{r depends, echo=FALSE, warning=FALSE, include=FALSE}

library(rstan)
library(rethinking)
library(bayesplot)
library(tidyverse)
```


# Abstract

The timing and duration of pollen shed and cone receptivity in lodgepole pine affects fecundity, gene flow, and adaptation in the species. Being able to predict the timing of pollen shed and cone receptivity ("flowering") will help us understand patterns of local adaptation, gene flow, and population structure as well as the effects of climate change on forests. Using 15 years of data from 7 lodgepole pine provenances grown at 7 sites, we built a multilevel Bayesian models of lodgepole pine pollen shed and cone receptivity phenology and used it to determine the amount of forcing required to begin and end the flowering period for both male and female strobili. With this method, any record of mean daily temperature can be used to estimate lodgepole pine flowering phenology. Cones become receptive slightly after pollen starts shedding and receptivity ends slightly before pollen shed ends. Provenance did not have a strong effect on flowering time. In simple climate change scenarios with increases of 2 and 5 degrees, the amount of phenological overlap between sites did not significantly increase or decrease with increasing temperatures because earlier flowering dates keep the rate of temperature accumulation within the flowering period relatively consistent.

```{r global, echo=FALSE}
forcingtype = "scaled_ristos"
```

```{r data, echo=FALSE, warning=FALSE}
# combined phenology and weather data 
phenology_data <- read.csv("../../data/phenology_heatsum_all.csv", header=TRUE, stringsAsFactors = FALSE) %>%
  filter(forcing_type==forcingtype)
phenology_data_model <- read.csv("../../data/phenology_heatsum.csv", header=TRUE, stringsAsFactors = FALSE)
# 
# seed orchard provenances
SPU_dat <- read.csv("../../../phd/data/OrchardInfo/LodgepoleSPUs.csv",
                    header=TRUE, stringsAsFactors = FALSE) %>%
  dplyr::select(SPU_Name, Orchard) %>%
  rename(Provenance=SPU_Name)


# join provenance and phenology data

dat <- phenology_data %>%
  na.omit() %>%
  left_join(SPU_dat) %>%
  unique()

```



# Outline intro
Pollination phenology is important because

- gene flow, dispersal - spatial and temporal variation
- reproduction, demography
- assisted migration, conservation
- seed orchard operations, commercial/agricultural/breeding

While vegetative phenology has been the focus of much research, pollination phenology is relatively unexplored in conifers. The focus is usually on day of year and comparing provenances, looking for local adaptation. They aren't predictive. 

- examples

Benefits of predictive/mechanistic models

- climate change
- predict out of dataset

Predictive models are scarce because they rely on excellent phenological records with high quality temperature datasets.

- counterexamples with strengths and weaknesses

Such a record does exist for lodgepole pine in British Columbia.

- BC seed orchards, collection rationale, existing dataset

Understanding the precise relationship between temperature and pollination phenology is valuable in lodgepole because

- it's an economically important species with lots of replanting and we need to understand how populations we're planting on the landscape will interact with local populations
- lots of phenotypic and genomic data available in lodgepole pine. Local adaptation, population structure. Understanding temporal and spatial variation in pollination phenology could help disentangle effects on population differentation (local adaptation, gene flow, etc) and explain population structure.

In this paper, we build a predictive model of pollination phenology in lodgepole pine that estimates the amount of forcing required to cause lodgepole pine to begin and end flowering. Using these estimates, any record of mean daily temperature can be used to predict the timing and duration of flowering in lodgepole pine.

We confirm that pollination phenology in lodgepole pine is not strongly influenced by provenance, then used the model results to calculate variation in the timing and length of phenological period at 7 sites in British Columbia between 1997 and 2011. Last, we considered how climate change may affect pollination phenology.

# Introduction
As the climate changes, spring phenological events like budburst and flowering will advance, especially for plants active in rapid seasonal transitions and short growing seasons [@Pau2011], like many high elevation and latitude conifers. 
This effect is already obvious in many species [@Parmesan2006; @Franks2014]. 
Changes in pollination phenology can affect fecundity, gene flow, and even range size in a species and have effects [VAGUE] on dependent species [@Inouye2008; @Chuine2001b].


Conifers are a big part of the enormous northern hemisphere forests and they have wide ranges with lots of local adaptation. Common garden experiments and genetic work reveal extensive local adaptation in many forest tree species, especially boreal and temperate conifers (reviewed in @Alberto2013). A locally adapted population only grows optimally in a subset of the range and may tolerate a more limited climatic range than the species as a whole. In northern hemisphere conifers, local adaptation often reflects strong trade-offs between avoidance of cold damage and competitive height growth (summarized in @aitken2008). 

Coniferous forest trees are wind pollinated wtih pollination possible over large distances. [EXAMPLES]
Pollen is shed from male strobili and must arrive at receptive female strobili for successful pollination .rpe
Shifts in the timing of pollen shed and cone receptivity (pollination phenology) in conifers could lead to gene flow changes that hinder or promote adaptation under climate change, decrease fitness, and even affect reforestation via seed production declines. 
[Also affects gene flow now and is important for understanding current spatial genetic structure and local adaptation] 

Lodgepole pine is a good representative of these issues - an economically and ecologically important tree species facing multiple threats from climate change [@Schneider2010; @Sambaraju2012; @Hamann2006]. 
Lodgepole pine has a very large geographical distribution (across 33$^\circ$ latitude and 31$^\circ$ longitude) encompassing a wide range of climates and soils (Fig. \@ref(fig:rangemap) with widespread and significant local adaptation in many traits. 
For example, populations from both northern interior British Columbia and northern Idaho can survive in areas with mean annual temperatures between -4 and 6 $^\circ\mathrm{C}$, but the northern British Columbia population survives best where mean annual temperatures are ~ 1 $^\circ\mathrm{C}$ and the Idaho population best at ~ 4 $^\circ\mathrm{C}$ [@Rehfeldt1999a]. 
Local adaptation in lodgepole pine can be observed even at relatively small spatial scales when topographic variability is high: in a reciprocal transplant experiment, growth declines were observed when moving high elevation populations just 100m in elevation [@rehfeldt1983]. [Summarize/cite adaptree work]


Pollen is an important vector for identifying gene flow in lodgepole pine because outcrossing is common, pollen dispersal is extensive, and seed dispersal is relatively limited [@Ennos1994]. 
There is evidence for spatially varying levels of gene flow in the species as populations from areas with higher regional climate heterogeneity have higher genetic variance [@Yeaman2006]. 
Pollination phenology could control this. 

Pollination phenology determines which populations can exchange genes, but work on predicting the timing of pollen shed and cone receptivity is scarce and coarse.
Determining the amount of forcing required for a phenological event to occur is rarely done for species that lack economic importance because collecting the data is too much work. 
When it is done, estimates are often location and population specific. Pollination phenology examples are uncommon for conifers. Simple heat accumulation thresholds (*Pinus taeda*) [@Boyer1978] or elevation (*Pinus flexilis*) [@Schuster1989] were used previously to explain or predict pollen shed in limited spacial and temporal contexts. Risto Sarvas's work in Scots pine is a rare exception. [Talk about Sarvas investigations for mechanistic background, why using temperature, etc.] 

@owensReproductiveBiologyLodgepole2006 reports that lodgepole pine pollen shed and cone receptivity occur when degree days reach about 500 at a threshold of 5 $^\circ\mathrm{C}$, but this is the only report of pollination phenology prediction we could find for lodgepole pine and limited details are provided. Models of lodgepole pine vegetative phenology, on the other hand, are better represented in the literature (*e.g.* @Chuine2001), and pollen shed and cone receptivity are not expected to have additional or more complex triggers or model forms than budburst [@Chuine2013a]. Plus we can rely on Sarvas's mechanistic work, which Chuine uses in her work.

Predicting pollination phenology will also have practical benefits. SMP

<!--Seed orchard managers in British Columbia are particularly concerned about protandry, when all pollen in an area is shed before cones become receptive (*personal communication, Chris Walsh, Former Seed Orchard Manager, Kalamalka Seed Orchards, February 13, 2013*). 
Protandry occurs in particularly hot and dry years [@Owens2005]. If this pattern holds, protandry could become more common in natural populations, leaving some populations pollen limited and likely hampering local adaption. [Outcrossing! Inbreeding!] But it looks like this really only happens at the *individual* level and not the *population* level and isn't something I need to be concerned about. -->

This paper relies on 15 year pollination phenology data set collected in British Columbia lodgepole pine seed orchards. 
Seed orchards produce seed for reforestation. 
While not set up as common gardens, several provenances are typically represented at each  site allowing testing for provenance effects and genetic x environmental interaction. and genotypes usually appear multiple times within a site and sometimes at multiple sites.

We use pollination phenology and temperature data to fit a model predicting pollen shed and cone receptivity in *Pinus contorta* var. *latifolia*.

Specifically, we answer 1) What is the relationship between temperature and pollen shed and temperature and cone receptivity timing and length? 2) Does provenance affect that relationship? 3) How does pollination phenology vary between cold and hot years? 4) Will protandry become more common?

[EXPLAIN WHY THE HECK I'M LOOKING AT PHENOLOGICAL OVERLAP BETWEEN THESE SITES. It's a simplification of what I'm doing in chapter 2 and gives us some insights into climate change and flowering phenology in lodgepole.]

# Methods

My aim was to model pollination phenology in lodgepole pine and calculate synchrony across the distribution. 
To determine the timing of pollen shed and cone receptivity, we modeled the forcing requirements for pollination phenology in lodgepole pine and investigated differences in forcing requirements between males and females and among provenances. 
We then compared the overlap in flowering periods between males and females across sites.

## Study species

Lodgepole pine is a wind-pollinated monecious conifer with an extensive range and well documented local adaptation [CITE]. While lodgepole pine has four subspecies, this work concerns only *Pinus contorta* subsp. *latifolia*. Pollen and seed cone buds differentiate in late summer and early fall, then go dormant. As temperatures warm the following year, buds resume development and strobili "flower"; receptive female strobili exude pollination drops between bracts that capture pollen shed from mature male strobili.

```{r rangemap, fig.cap='(ref:rangemap)', out.width='100%', echo=FALSE}
knitr::include_graphics('../../../phd/maps/SpeciesDistribution/distmap_pincon.png')
```

### Data

#### Phenology Data

This project takes advantage of an existing lodgepole pine pollination phenology dataset collected over a decade and a half by government and industry workers in seed orchards in British Columbia. Seed orchards produce large amounts of tree seed for reforestation from parent trees sourced from provenances around the province. To plan for future seed production and orchard establishment and management, seed orchard managers monitored pollination phenology and seed production at many of the orchards. Pollination phenology data was collected at the Prince George Tree Improvement Station in British Columbia beginning in 1997 and collection at many other BC tree orchard sites began in 2006 under the Forest Genetics Council's Operational Tree Improvement Program 0722 [@webberCollectionCropStatistics2007]. Collection continued intermittently through 2012.


##### Sites
Trees selected from across the British Columbia portion of the lodgepole pine range are grown in seed orchards as part of tree breeding and seed production programs. Between 1997 and 2011, flowering phenology of lodgepole pine was recorded at 7 seed orchard sites in the interior of British Columbia. We contacted Seed Orchard Managers and other forestry professionals across British Columbia in 2012 and received pollination phenology data from C. Walsh, previously at Kalamalka Seed Orchards (now retired), R. Wagner at the Prince George Tree Improvement Station (ALSO RETIRED?), and J.E. Webber previously at the Glyn Road Research Station (now retired). 4 of the sites are clustered near to one another around Vernon, BC. Sites span about 5 $^\circ$ of latitude and are at elevations from 466 to 638 m.

```{r sitemap, fig.cap='(ref:sitemap)', out.width='100%', echo=FALSE}
knitr::include_graphics('../../../phd/maps/SiteMaps/sitemapwithPCIC.png')
```

[TABLE OF SITE ELEVATION AND LAT/LON - here or in suppl/data]

##### Provenances
<!--
I obtained pollination phenology records from 17 of the 26 lodgepole pine seed orchards in British Columbia. Thirteen out of 17 orchards in my data set are first generation orchards and should faithfully represent their provenances. These first generation orchards represent 6 provenances at 5 sites. Second generation orchards have been selectively crossed and this may skew the mean or variance of phenology for a provenance if pollination phenology varies by provenance.-->

Parents of trees grown at the seed orchard sites were sourced from 7 Seed Planning Units (SPUs) (Fig. \@ref(fig:spu). SPUs are organizational units in BC forestry that are based on biogeoclimate regions. Trees with the same SPU provenance are grown together in an orchard at a given site. Genotypes (labelled with a Clone number in the data) in the orchards are represented by multiple ramets. 

Most provenances are represented at 2 to 3 sites and have at least three years of data at a given site spanning 1997-2012 (Tab. \ref{contingency}). The Prince George Tree Improvement Station (PGTIS) provides a continuous 15-year record of its three orchards' phenology. Most sites only observe one ramet per clone; PGTIS typically observed 2-4 ramets per clone.

![Contingency table of years of data for Seed Planning Units (rows) and Seed Orchard Sites (columns). Seed Planning Zones, used as provenances in this project, are usually represented at multiple years and multiple sites. There is particularly good representation at PGTIS. \label{contingency}](../../../phd/phd_proposal/figure/contingencytable.png)

```{r spu, fig.cap='(ref:spu)', out.width='100%', echo=FALSE}
knitr::include_graphics('../../../phd/maps/SPUmaps/SPUmap_subset.png')
```

[table with Site Columns and SPU rows with years of data as values]
 [table of number of trees/clones in a given year for a given site/spu combo]

##### Phenology scoring protocol
Protocol C in [@Woods1996] was used as the basis for collecting pollen shed and cone receptivity data, though operational constraints led to some modifications. Workers monitored seed orchards for the beginning of pollen shed and cone receptivity. Approximately, 15 clones, usually represented by 2 trees each, were selected for specific observations. When the active period seemed to be starting, workers went into the orchard every few days to make observations on the selected trees.

Stages of pollen and seed cone development are described by @owens1984 and updated in @owensReproductiveBiologyLodgepole2006 and were used as a general guide for determining the phenological state of pollen and seed cones. Pollen cones are flowering when tapping causes pollen to be released and seed cones are flowering when there are gaps between the bract-scale complexes. Pollination drops are also produced during flowering, though they recede midday if pollen is not present [@owensReproductiveBiologyLodgepole2006]. 

"Flowering" states were recorded for both pollen shed and cone receptivity at the level of each tree. Protocol C recommends marking the dates when 20% of the cones on a tree have begun flowering and when 80% of the cones on a tree have finished flowering. Operationally, there was some subjectivity and tree-level states for each cone type should be interpreted as "starting flowering" and "finished flowering."  

[DESCRIBE SURVEY PERIODS]
<!--```{r spu, fig.cap='(ref:spu)', out.width='100%', echo=FALSE}-->
```{r surveys, fig.cap='(ref:surveys)', fig.height=12, out.width='100%', echo=FALSE,}
# Index by Site, Provenance, and Year
dat$spyind <- group_indices(dat, Site, Year, Provenance)

surveydf <- select(dat, spyind, Year, Site, Provenance, DoY) %>%
  distinct()

ggplot(surveydf, aes(x=DoY, y=as.factor(spyind), color=Provenance)) +
  geom_point(pch=3) +
  geom_line() +
 facet_grid(rows=vars(Site,Year), scales="free_y") +
  scale_color_viridis_d(option="C") +
  scale_shape_manual(values=c(1:7)) +
  theme_bw(base_size = 15) +
  theme(strip.text.y = element_text(angle = 0), axis.text.y = element_blank(), axis.ticks = element_blank(), axis.title.y = element_blank(), panel.border=element_blank(), panel.grid.minor.y=element_blank(), panel.grid.major.y = element_blank()) +
  ggtitle("Observation Dates")
```

Observations were not made every day and survey periods varied in length (Fig. \@ref(fig:surveys)). Not all trees have complete phenological records, start or end dates may be unknown.


##### Harmonization and cleaning

There were some differences in how data was recorded at the Prince George Tree Improvement Station versus the other sites. For most of the Prince George data, trees were marked as flowering or not flowering on each day of observation. At other sites, only the first day observed flowering and the first day observed finished flowering were recorded. At sites using that minimal recording schema, we inferred states for many clones for some observation dates at the orchard. *E.g*, if tree A and B are in the same orchard and A is observed to be receptive on Monday and tree B on Tuesday, then we assumed that tree B was not yet receptive on Monday. 

We cleaned and harmonized the data for analysis in a single model using R scripts [PROVIDE] so that, for pollen shed and receptivity, stage 1 = not yet flowering, stage 2 = flowering, and stage 3 = finished flowering (Tab. \@ref(tab:phenophasederivation)). 

Phenophases in the field were recorded using different symbol sets and resolutions. We assigned each symbol to one of the three phenophases above. The few trees that did not produce cones were assigned phenological stage 0 and were not included in the analysis.

In total, there were `r nrow(phenology_data)` usable observations of phenological state.

```{r phenophasederivation, echo=FALSE}
no_flowers <- '0'
#before_flowering <- c('1', '2.5', '-') #stage 1
before_flowering <- c("1, 2.5, -")
#flowering <- c('3', '3.5', '4', '4.5', '5', 'pollenshed20', 'receptive20') #stage 2
flowering <- c("3, 3.5, 4, 4.5, 5, pollenshed20, receptive20")
#after_flowering <- c('-', 'receptive80', 'pollenshed80') #stage 3
after_flowering <- c("-, receptive80, pollenshed80")
derived_phenophase <- c(0:3)
male_desc <- c("none produced", "not yet shedding", "shedding", "finished shedding")
female_desc <- c("none produced", "not yet receptive", "receptive", "no longer receptive")
recorded_as <- c(no_flowers, before_flowering, flowering, af=after_flowering)

phenophase_legend <- data.frame(Phenophase = derived_phenophase, Symbols = recorded_as, Male.Cones = male_desc, Female.Cones = female_desc)
knitr::kable(phenophase_legend, caption = "Phenophases, symbols used to record them in the raw data, and phenophase definitions")
```

[Figure showing phenology data somehow]

#### Weather data

Daily weather data at seed orchard sites was extracted from PNWNAmet, a daily gridded meteorological dataset at 1/16 $^\circ$ (~ 6km) over northwest North America  [@pacificclimateimpactsconsortiumPNWNAmet194520122014]. The closest point in the PNWNAmet grid was used for each site (Fig. \@ref(fig:sitemap)). Mean daily temperature was calculated as the average of the minimum and maximum daily temperatures. 

Gridpoint data was adjusted using monthly site data. As gridpoints did not align exactly with site locations and there were significant elevation differences between many of the sites and gridpoints, PNWNAmet data did not accurately represent the temperature at sites, so a correction factor was applied. First, mean monthly temperatures for the sites were generated using the ClimateNA v6.20 software package [@wang2016]. Monthly site specific data then was regressed on monthly PNWNAmet data to determine the correction factor for daily PNWNAmet data. 

[MOVE MOST OF THE REST OF THE SECTION TO SUPPLEMENTAL]
For each site, climateNA temperatures $T$ were regressed on mean monthly PNWNAmet temperatures $t$ with `lm` in R's `stats` package (Tab. \@ref(tab:climatenacorrection-pars)).

\begin{equation} 
  T = \alpha + \beta t 
  (\#eq:tempadjust)
\end{equation} 

<!--Table: (\#tab:label) Caption here-->
Table: (\#tab:climatenacorrection-pars) Table of intercepts $\alpha$ and slopes $\beta$ for each site.

|            |  intercept|     slope|Site        |
|:-----------|----------:|---------:|:-----------|
|Kalamalka   |  0.2052868| 0.9820646|Kalamalka   |
|KettleRiver |  1.0375494| 1.0353338|KettleRiver |
|PGTIS       |  0.3196696| 0.9888230|PGTIS       |
|PRT         |  0.3726752| 0.9805055|PRT         |
|Sorrento    |  1.9729000| 0.9950774|Sorrento    |
|Tolko       | -0.0164089| 0.9698295|Tolko       |
|Vernon      |  0.2035381| 0.9839158|Vernon      |

For each site, daily mean temperature  was estimated by 

\begin{equation} 
T_{site, date} = \alpha_{site} + \beta_{site} t_{site, date}
(\#eq:tempcorrect)
\end{equation}
where $t$ are the mean daily temperatures extracted from PNWNAmet.

#### Forcing units

Flowering is a developmental process that speeds up and slows down according to the current temperature. Forcing units describe the relative effect of temperature on development. Observable phenological events occur only after a certain amount of forcing has accumulated. The mean daily temperature $t$ was mapped to forcing units with a function that describes the relationship between temperature and development rate.

\begin{equation}
R(t) = \frac{1}{1+ e^{-0.185*t-18.4}}
(\#eq:ristocalc)
\end{equation}

This equation for forcing units (Eqn. \@ref(eq:ristocalc)) is based on experimental work in temperate forest tree species [@Sarvas1972, @hanninenModellingBudDormancy1990] and was verified to perform better than growing degree day for phenology estimation [@Chuine1999].

Accumulated forcing units on day $d$ ($f(d)$) are the sum of the relative temperature effect from January 1 (ordinal date $1$) to day $d$.

\begin{equation}
f(d) = \sum_{i=1}^d R_d(x)
(\#eq:accumulatedforcingcalc)
\end{equation}

Cooler sites, like Prince George, accumulate forcing slower than warmer sites like Kalamalka. 

[ACCUMULATED FORCING UNITS FIGURE]

## Phenology Modeling 

We modeled phenological states as a function of accumulated forcing units with separate Bayesian multilevel ordinal logistic models for male and female strobili. A logistic cumulative link function accounts for the ordering of phenological states and relates phenological states to a linear model. This type of model makes no assumption about the distance between phenological states. The model parameters describe a probability function for each state. Cutpoints $\kappa$ separate each state while the slope parameter of the linear model influences the steepness of the curves. Transitions between phenological states happen rapidly when the slope is large and slowly when it is small. 

The likelihood of being in state $s \in \{1,2,3\}$ is

\begin{equation}
\mathrm{Pr}(S=s)=\text{OrderedLogistic}(s~|~\eta,\kappa) = \left\{ \begin{array}{ll} 1 -
\text{logistic}(\eta - \kappa_1)  &  \text{if } s = 1 \\[4pt]
\text{logistic}(\eta - \kappa_1) - \text{logistic}(\eta - \kappa_2)  &
\text{if } s=2  \\[4pt] \text{logistic}(\eta -
\kappa_2) - 0  &  \text{if } s = 3 \end{array} \right.
(\#eq:likelihood)
\end{equation}

where the cutpoints $\kappa_s < \kappa_{s+1}$ and $\eta$ is a linear model.
\begin{equation}
\begin{array}{rlr}
S_i & \sim \mathrm{OrderedLogistic}(\eta,\kappa) & \text{probability of data}\\
\eta_i & = (\beta + \beta_{site} +\beta_{prov} + \beta_{clone} + \beta_{year}) f_i  & \text{linear model}\\
\kappa_k & \sim \mathrm{gamma}(20,1) & \text{fixed priors}\\
\beta & \sim \mathrm{exponential}(2) & \text{}\\
\beta_{site} & \sim \mathrm{normal}(0,\sigma_{site}) & \text{adaptive priors}\\
\beta_{prov} & \sim \mathrm{normal}(0,\sigma_{prov}) & \text{}\\
\beta_{year} & \sim \mathrm{normal}(0, \sigma_{year}) \\
\beta_{clone} & \sim \mathrm{normal}(0, \sigma_{clone}) \\
\sigma_{site} & \sim \mathrm{exponential}(5) & \text{hyperpriors}\\
\sigma_{prov} & \sim \mathrm{exponential}(5) \\
\sigma_{year} & \sim \mathrm{exponential}(5) \\
\sigma_{clone} & \sim \mathrm{exponential}(5) \\
\end{array}
(\#eq:modelequations)
\end{equation}

The linear model $\eta$ includes a population mean slope $\beta$ with deviations from that for site, provenance, clone and year. Priors were designed to force the correct sign (positive) and order of magnitude [@lemoineMovingNoninformativePriors2019]. Priors on the site, provenance, clone, and year effects are adaptive and effects are estimated from the data with partial pooling. 

The model was fit in the probabilistic programming language Stan (vers. 2.19.2 via RStan [@standevelopmentteam2019]), which uses the No-U-Turn Sampler, an efficient Markov Chain Monte Carlo method that extends the Hamiltonian Monte Carlo algorithm [@carpenter2017], to sample the joint posterior. We ran 4 independent chains for 4500 iterations, discarding 1000 warm-up iterations for a total of 14,000 posterior samples for each parameter. Model convergence and performance were considered good; $\hat R$ was $< 1.01$ and bulk effective sample size was more than 100x the number of chains for all parameter values (Female model: Tail ESS 2,231-11,286, median 9,326; Bulk ESS 1,434-31,750, median 11,059. Male model: Tail ESS 2,843-11,332, median 9,371; Bulk ESS 1,617-30,793, median 11,633) [@vehtari2019]. There were no divergences. Visual inspection of energy plots and rank plots showed acceptable sampling behavior.

#### Model Assumptions 
We assume that chilling requirements are always met and that chilling and forcing periods do not overlap. Forcing is a function of temperature only and does not incorporate photoperiod or chilling. Transitions between each state occur at the same rate, *i.e.* $\beta$ parameters cannot vary by phenological state. This model does not account for damage and abnormal development from very cold and very warm temperatures. [Cite Sarvas's weird hot stuff and some late spring phenology] We assume that forcing does not begin to accumulate until January 1.

#### Flowering period timing and length

We defined the beginning of the flowering period as the point at which a tree was 20% likely to have passed out of stage 1 and the end of the flowering period as the probability at which a tree was 80% likely to have passed out of stage 2. This is equivalent to 20% of the trees in the population having reached at least stage 2 and 80% having reached stage 3.

##### Begin and end of flowering period
The accumulated forcing units required to reach the beginning of the flowering period, $f_{begin}$, is 
\begin{equation} 
\begin{array}{rl}
\mathrm{Pr}(s > 1) = 0.2 & = \mathrm{logistic}((\beta + \beta_{site} +\beta_{prov} + \beta_{clone} + \beta_{year})f - \kappa_1) \\
f_{begin} & = \frac{\mathrm{logit}(0.2) + \kappa_1}{\beta + \beta_{site} +\beta_{prov} + \beta_{clone} + \beta_{year}}
\end{array}
(\#eq:fbegin)
\end{equation}


The accumulated forcing units required to reach the end of the flowering period, $f_{end}$, is, simularly 

\begin{equation} 
\begin{array}{rl}
\mathrm{Pr}(s > 2) = 0.8 & = \mathrm{logistic}((\beta + \beta_{site} +\beta_{prov} + \beta_{clone} + \beta_{year})f - \kappa_2) \\
f_{end} & = \frac{\mathrm{logit}(0.8) + \kappa_2}{\beta + \beta_{site} +\beta_{prov} + \beta_{clone} + \beta_{year}}
\end{array}
(\#eq:fend)
\end{equation}

[SHOW toy FIGURE FOR THIS?]

We calculated $f_{start}$ and $f_{end}$ for all combinations of clone, site, provenance, and year that occur in the dataset for all samples from the posterior. The range of forcing units that stage 2 occurs over, *i.e.* the flowering period, is 

\begin{equation}
f_{range} = f_{end}-f_{begin}
(\#eq:frange)
\end{equation}
[THIS EQUATION IS OVERKILL]

##### Translating between accumulated forcing units and days
While $fbegin$ and $fend$ are constant for a given combination of clone, site, provenance, and year, $dbegin$ and $dend$ differ from year to year and location to location. For years and locations of interest, accumulated forcing units were calculated from mean daily temperatures and used to translate between accumulated forcing units and day of year. The day of year that the accummulated forcing unit reached at least $f_{begin}$ or $f_{end}$ was $d_{begin}$ or $d_{end}$. The length of the phenological period was then calculated as

\begin{equation}
length=d_{end}-d_{begin}
(\#eq:drange)
\end{equation}

#### Posterior predictive checks [PROB SUPPLEMENTAL, JUST ADD LINE THAT THEY'RE OK]

To check the realism of the model, the observations of phenological states were compared to model predictions. When the model was fit in Stan, we also simulated one phenological state from all model configurations for each phenological observation in the dataset. The probability of having reaching a given phenological state at a given forcing accumulation predicted by the model matched the data well (Fig. \@ref(fig:ppccum)). 

```{r ppccum, fig.cap='(ref:ppccum)', out.width='100%', echo=FALSE}
knitr::include_graphics('../../../phd/LabNotebook/images/2019-10-27ppc_cumprop.png')
```

```{r ppcprop, fig.cap='(ref:ppcprop)', out.width='100%', echo=FALSE}
knitr::include_graphics('../../../phd/LabNotebook/images/2019-10-28ppcstatesexprop.png')
```

The proportion of state 2 observations (flowering) was predicted correctly by the model [X%] of the time for females and [Y%] for males (Fig. \@ref(fig:ppcprop)). Flowering is predicted with greater accuracy than the other phases and male flowering is predicted better than female flowering. Phases 1 and 3 are likely predicted less well because the observation dates (and associated accumulated forcing) are likely to be very close to the transitions into or out of state 2, *i.e.*, the most difficult ranges of forcing units over to predict phenological state.

[WHAT DOES THE SITE GRAPH ADD?]

[IF I KEEP THIS GRAPH, ORDER BY N-S OR SOMETHING OTHER THAN ALPHA]

```{r ppcsite, fig.cap='(ref:ppcsite)', out.width='100%', echo=FALSE}
knitr::include_graphics('../../../phd/LabNotebook/images/2019-10-28ppcsite.png')
```

When the model incorrectly retrodicts a phenophase, it almost always predicts an adjacent phenophase. [UNNECESSARY, DROP?]

```{r ppccounts, fig.cap='(ref:ppccounts)', out.width='100%', echo=FALSE}
knitr::include_graphics('../../../phd/LabNotebook/images/2019-10-28ppcstatesexcount.png')
```


<!--
The model is very likely to correctly predict phase 1 and 3 at higher and lower amounts of accumulated forcing and is less accurate near the transitions into and out of stage 2.

![Proportion of observations correctly predicted as forcing accumulates \label{ppc_forcing}](../../phd/LabNotebook/images/2019-10-28ppcforcingphase.png)-->


#### Average predictive comparisons 
Average predictive comparisons allow parameters in non-linear models to be compared like regression coefficients in linear models (@gelman2007, @shirley2015). An APC for the site effect, for example, describes how much the accumulated forcing required to begin flowering would change if trees were grown at sites other than the one they were actually grown at. If site isn't important, then changing the site shouldn't make a difference and the APC will be small. The average predictive comparisons for site, provenance, clone, and year were calculated to determine their relative influence on $f_start$.

Average predictive comparisons for each effect $u$ were calculated by calculating the absolute value of the difference between the expected value at a given data point $n$ (where $n$ were all uniquely occuring combinations of site, provenance, clone, and year in the phenology dataset) for the input variable of interest $u^{(k)}$ and the other input variables $v$ and that datapoint for its actual value of $u$ (Eqn. \@ref(eq:apc)). That difference was multiplied by a weight $\omega$ representing the similarity in $v$ between a datapoint with $u_i$ versus $u^{(k)}$. The weighted average of these differences is taken over all data points, all values of $u$, and 1400 samples from the posterior distribution. 

\begin{equation}
\hat \Delta_u = \frac{\sum_{i=1}^{n}\sum_{k=1}^{K}\sum_{s=1}^S[\sum_{j\in\{k\}}\omega_{ij}]|\mathrm{E}(y|u^{(k)},v_i, \theta^{(s)})-\mathrm{E}(y|u_i,v_i, \theta^{(s)})|}
{\mathrm{S}\sum_{i=1}^{n}\sum_{k=1}^{K}[\sum_{j\in\{k\}}\omega_{ij}]}
(\#eq:apc)
\end{equation}

Comparison calculations are weighted ($\omega$) to reflect the distribution of data, but the weighting function is not specified in @gelman2007. In the predcomps package, @chudzicki2018 implements a weighting function for ordered categorical and numerical variables based on Mahalanobis distance, but it is inappropriate for unordered categorical data, which all of the effects in this model are. We developed a weighting function that compares the elements of the other input variables $V_i$ in pairs of rows. Matches are assigned 1 and non-matches 0. Each $V_i$'s matches are summed and then divided by 4, creating a weight for each row x row comparison, which makes an $n_i$ x $n_i$ matrix of weights. The overall weight $w$ for a given $V_i$ is the normalized sum of the weights from comparing it to all other rows (@chudzicki2018). 

APCs were calculated in terms of both forcing and days. 20 timeseries of weather data from the sites were chosen, spanning coolest to warmest, to calculate the APCs in days.

#### Model predictions

Specific predictions for the start and end of the phenological period can be made from the model for sites, years, clones, and provenances using the parameter of interest. General predictions can also be made using the mean parameter for site, year, clone, and provenance along with the estimated $\sigma$ for that parameter. We made general predictions from the model for $fstart$ and $fend$ by taking 7000 samples from the posterior then simulating 30 values for site, provenance, year, and clone $\beta$s from normal distributions with the mean parameter effect across each cluster and $\sigma$ for each cluster. We then used the equations above for $fstart$ and $fend$. To make predictions in terms of days, We translated $fbegin$ and $fend$ into day of year in 15 years at the 7 seed orchard sites.

#### Validation

The male model was validated using pollen slide data at several of the orchards.
We also compared the general model predictions to the actual data.

#### Length

#### Overlap

##### Overlap with climate change
Simulated climate change/warming by adding 2 degrees to the mean temp every day and 5 degrees.

#### Flowering period date

# Results

## Model parameters - estimates

### fstart and fend  overall - estimates

Receptivity requires more forcing to begin and less to end than pollen shed (Fig. \@ref(fig:fstartfend)). [Add 50% HPDI numbers]

```{r fstartfend, fig.cap='(ref:fstartfend)', out.width='100%', echo=FALSE}
knitr::include_graphics('../../../phd/LabNotebook/images/2019-11-25_forcingstartend.png')
```


## Interpreting Effects: Average Predictive Comparisons

Differences in clone or year cause larger changes in $fstart$ than site or provenance (Fig. \@ref(fig:apcforce)).

[FIX FIGURE SO NON-OVERLAPPING]

```{r apcforce, fig.cap='(ref:apcforce)', out.width='100%', echo=FALSE}
knitr::include_graphics('../../../phd/LabNotebook/images/2019-11-13_apc_forcing_spacedyears.png')
```


$fstart$ was translated into day of year for 30 years (\@ref(fig:apcday)). Differences within effects almost never cause more than a difference of 3 days. Site and provenance rarely change a prediction by even a day. Year and clone have the largest effects and the effect of clone is especially large for cone receptivity (1-3 days vs 1-2 for males). 

```{r apcday, fig.cap='(ref:apcday)', out.width='100%', echo=FALSE}
knitr::include_graphics('../../../phd/LabNotebook/images/2019-11-13_apc_days_spacedyears.png')
```

We examined the variability of the start date across the range ($t_1$) to see if it shrinks in warmer years and as the climate changes. We calculated overlap between within population cone receptivity and pollen shed under current conditions and with climate warming and see if the heat sum requirements for cone receptivity and pollen shed are different.


## Model summary and parameter estimates [SUPPLEMENTAL PROBABLY ]

```{r model data, eval=FALSE, echo=FALSE}
ffit.stan <- readRDS("../2019-10-28phenologyFEMALE.rds")
mfit.stan <- readRDS("../2019-10-28phenologyMALE.rds")

#need to drop noncentered parameters. model_examination.R is more uptodate

fsum <- rstan::summary(ffit.stan)$summary
fsum <- as.data.frame(fsum)
farray <- as.array(ffit.stan)

fsum <- rstan::summary(ffit.stan)$summary
fsum <- as.data.frame(fsum)
fsum$par <- rownames(fsum)
fpost <- as.data.frame(ffit.stan) %>%
  select(-contains("z"))
fparam_names <- colnames(fpost)


msum <- rstan::summary(mfit.stan)$summary
msum <- as.data.frame(msum)
msum$par <- rownames(msum)
mpost <- as.data.frame(mfit.stan) %>%
  select(-contains("z"))
mparam_names <- colnames(mpost)

#mf
fpost$sex <- 0
mpost$sex <- 1

post <- full_join(mpost, fpost) 
```

<!--Clones not included-->

*Female*
```{r paramfemaletable, eval=FALSE, echo=FALSE}
knitr::kable(
fsum[str_detect(fsum$par, "b_clone", negate=TRUE) , ], digits=3, 
caption = "Female model parameters")
```

*Male*
```{r parammaletable, eval=FALSE, echo=FALSE}
knitr::kable(
msum[str_detect(msum$par, "b_clone", negate=TRUE) , ], digits=3,
caption = "Male model parameters")
```

```{r compare plot function, echo=FALSE, eval=FALSE}

compare_fm <- function(femplot, mplot, nrow = 2, ...) {
    bayesplot_grid(
        femplot, mplot,
        grid_args = list(nrow = nrow),
        subtitles = c("Female",
                      "Male"),
        ...
    )
}

compare_fm_wide <- function(femplot, mplot, ncol = 2, ...) {
    bayesplot_grid(
        femplot, mplot,
        grid_args = list(ncol = ncol),
        subtitles = c("Female",
                      "Male"),
        ...
    )
}
```


```{r compare site and prov intervals, echo=FALSE}
# ggplot(post, aes(x=beta, fill=as.factor(sex))) +
#     geom_density(alpha=0.5) + 
#     ggtitle("male and female speed of transition estimates")
# 
# fint_beta <- mcmc_areas(fpost, regex_pars = "beta") + ggtitle("beta")
# mint_beta <- mcmc_areas(mpost, regex_pars = "beta")
# 
# compare_fm(fint_beta, mint_beta, xlim=c(0.05, 0.10))
# 
# fkap <- mcmc_areas(fpost, regex_pars = "kappa")
# mkap <- mcmc_areas(mpost, regex_pars = "kappa")
# 
# compare_fm_wide(fkap, mkap, xlim=c(18, 29))
# 
# color_scheme_set("blue")
# fint_site <- mcmc_intervals(fpost, regex_pars = c("site", "prov"))
# mint_site <- mcmc_intervals(mpost, regex_pars = c("site", "prov"))
# 
# compare_fm_wide(fint_site, mint_site) 
# 
# fa_sigma <- mcmc_areas(fpost, regex_pars= "sigma") + ggtitle("variance")
# ma_sigma <- mcmc_areas(mpost, regex_pars= "sigma")
# 
# compare_fm_wide(fa_sigma, ma_sigma)
# 
# fa_y <- mcmc_areas(fpost, regex_pars = "year") + ggtitle("year")
# ma_y <- mcmc_areas(mpost, regex_pars = "year")
# 
# compare_fm_wide(fa_y, ma_y)

```

```{r compare clones, echo=FALSE}
# fint_c <- mcmc_intervals(fpost, regex_pars = c("clone"), point_est="none") + ggtitle("Clone effects")
# mint_c <- mcmc_intervals(mpost, regex_pars = c("clone"), point_est="none")
# 
# compare_fm_wide(fint_c, mint_c)
# ```
# 
# ### Sex diff
# 
# ```{r half transitions - no effects}
# fpost$f1 <- fpost$`kappa[1]`/fpost$beta
# fpost$f2 <- fpost$`kappa[2]`/fpost$beta
# mpost$f1 <- mpost$`kappa[1]`/mpost$beta
# mpost$f2 <- mpost$`kappa[2]`/mpost$beta
# 
# ff <- mcmc_areas(fpost, pars = c("f1", "f2")) + ggtitle("half transitions")
# mf <- mcmc_areas(mpost, pars = c("f1", "f2")) + ggtitle("half transitions")
# 
# compare_fm(ff, mf)
```
## Flowering period

Receptivity and pollen shed overlap for most of the flowering period within a year and site (Fig. \@ref(fig:floweringperiod). Pollen shed typically encompasses the entire receptivity period. Prince George, the most northern site, generally flowers later in the year than Kalamalka and has shorter flowering periods. Kettle River, the most southern site, but rather cool compared to Kalamalka, is in between.

```{r floweringperiod, fig.cap='(ref:floweringperiod)', out.width='100%', echo=FALSE}
knitr::include_graphics('../../../phd/LabNotebook/images/2019-11-28_5090doyhdpi.png')
```

## Length of phenology period

Flowering normally lasts between 10 and 20 days and receptivity is about 3 days shorter than pollen shed (\@ref(fig:floweringlength)) [GET REAL #S, DON'T JUST LOOK AT GRAPH]

```{r floweringlength, fig.cap='(ref:floweringlength)', out.width='100%', echo=FALSE}
knitr::include_graphics('../../../phd/LabNotebook/images/2019-11-28_lengthindays.png')
```

## Overlap now and in the future

Overlap between receptivity and pollen shed between sites depends on the timing and length of the flowering period at each site. Currently, PGTIS receptivity doesn't overlap with Kalamalka pollen shed and Kalamalka receptivity doesn't overlap with PGTIS pollen shed. PGTIS receptivity overlaps with Kettle River pollen shed longer than it does its own.

Increasing temperatures by 2 and 5 degrees does not cause large changes in flowering overlap between sites (Fig. \@ref(fig:overlap)). It may slightly shorten the length of overlap between pollen shed and receptivity within a site. It does lead to small amounts of overlap between Kalamalka's receptive cones and pollen shed at PGTIS, though PGTIS cones still aren't receptive when Kalamalka is shedding pollen. [RUN ALL THE SITES AND BE MORE QUANTITATIVE].

```{r overlap, fig.cap='(ref:overlap)', out.width='100%', echo=FALSE}
knitr::include_graphics('../../../phd/LabNotebook/images/2019-11-28_receptivityoverlap_warm.png')
```


# Discussion

## Conclusions regarding goals or hypotheses in intro

### Provenance differences


1) How strong are the genetic vs. environmental effects on pollination phenology? NOT, talk about APCs.

This means: we can use the same models across the entire range when predicting phenological events.

Discuss:
The budburst and spring pollination phenology of temperate tree species is determined by temperature, and there may be some regional variation in the temperatures required for pollen shed and cone receptivity;
in lodgepole pine, the threshold for shoot elongation was 5.1 $^\circ\mathrm{C}$ for coastal and interior provenances, but 6.5 $^\circ\mathrm{C}$ for northern provenances [@Chuine2001].

[UPDATE WITH IAN'S PUBLISHED 2017 STUFF] However, seedling spring vegetative phenology data from four growth chamber temperature and moisture treatments in the AdapTree project suggests that budburst shows substantial clinal variation (> 10% of phenotypic variance explained) only in the coldest treatment tested (MAT of 1 $^\circ\mathrm{C}$), and much colder than the seed orchard locations (Figure \ref{siteprovclimate}, Figure \ref{Liepe_Pheno_Corr}, @Liepe2014 (submitted)). Analysis of phenotypic data from an outdoor common garden in Vancouver (*personal commmunication, Ian Maclachlan, AdapTree Graduate Student, January, 1 2015*) also shows that very little variation exists among provenances for spring phenology, regardless of whether seedlings originate from wild-stand or selectively bred seed orchard seedlots.

[TALK ABOUT LIEPE?]

<!--![Variance in budbreak and budset for lodgepole pine explained by provenance climate. Seedlings from 281 locations across lodgepole's BC range were grown in growth chambers under four treatments: approximated seasonal and daily variation in temperature for geographic locations with a MAT of 1, 6, and 11 $^\circ\mathrm{C}$, with the warmest treatment having a dry and wet version. Pearson's correlation coefficient between phenotypic traits in lodgepole pine and provenance climates are presented. The vertical line represents the critical $r^2$ value after an adjustment for multiple inferences. From @Liepe2014 (submitted). \label{Liepe_Pheno_Corr}](../../../phd/phd_proposal/figure/Liepe_phen_corr.png)-->

2) How does faster spring warm-up like we expect under climate change affect within population mating success and frequency of outcrossing? 

[WOO EXCITING RESULTS I WAS WRONG! ORIGINAL EXPECTATIONS] We expect that faster spring warm-up will condense as well as advance growth initiation dates and also shorten pollination phenological periods by speeding development prior to and during the phenological period. This should further synchronize populations in similar climates and act as a barrier to breeding with populations in more different climates. On the other hand, it could favor outcrossing over selfing by causing protandry - pollen shed in advance of cone receptivity. Based on anecdotal reports from seed orchards managers, we expect protandry (and thus outcrossing) to increase under climate change.

ACTUAL Cooler sites have shorter flowering periods because they take longer to reach $fstart$ and by the time they do, it's pretty warm out and so flowering ends faster. Warmer sites, on the other hand, flower earlier in the year when temperatures during the flowering period are typically cooler, extending the flowering period.

This means that under climate change, assuming even heat up throughout the winter/spring (hah!), flowering period overlap between sites shouldn't change much or might even increase.

[FROST DAMAGE??]

[MAKE SURE TO COVER]

- Reflective analysis of scholarly work and conclusions in light of current knowledge in the field

### Compare heatsum to risto predictions
The limited reports of forcing required for lodgepole pine pollination use heatsum, typically with a threshold of 5 degrees C. They are difficult to directly compare because threshold not always reported nor is the start date for accumuluation.
@owensReproductiveBiologyLodgepole2006 reported heatsum of 500 with considerable variation, Wagner and Walsh x to y (Inc.2008). They are in the right ballpark for Walsh and Webber’s numbers (once divided by 24 for right units). However, W&W only start adding up heatsum in the month or so before pollination. We think the numbers are still roughly comparable because the 5 degree threshold in GDD5 calculations means that very little accumulates before then.

When converting observations/model 
- Comments on significance and contribution
- Comments on strengths and limitations

@Owens2001 on Western White Pine
> Degree-days are a useful way to predict seed cone receptivity and to a lesser extent pollen shed at one site in successive years but not between sites

### Limitations of the dataset

Benefits of this dataset include the large number of trees, long time series, multiple provenances grown at multiple sites giving a semi-common garden design, and the inclusion of clones. Limitations include interval and end censoring, especially at Prince George, irregular scoring systems, subjective scoring, non-random clone selection, and selective breeding.

### Grafting bias?

### Problems from breeding
Some orchards have been subject to selection that may have affected phenology. (TALK ABOUT IAN'S RESULTS?) There are two types of orchards: first generation and advanced. 
Each orchard contains trees that are the descendants of seeds and scions collected from mother trees in one particular provenance.
First generation orchards have not undergone selective breeding but may be subject to selection through the process of choosing mother trees (called "plus" trees by tree breeders), testing and selection of mother tree offspring, or culling of unfavorable trees in the orchard.
First generation orchard trees are clones of mother trees and their offspring grafted onto rootstock.
There are three types of first generation orchards subject to different levels of selection: 1.0, 1.5, and 1.75 generation orchards.
All orchards are subject to selection based on mother tree selection.
In 1.5 generation orchards, superior offspring of mother trees are selected for the orchard or poor trees are culled from the orchard. 
This selection may be strong; more than 50% of offspring can be rejected in this process due to poor growth form [@Ukrainetz2011].
In 1.75 generation orchards, data from 10 year tests of mother tree offspring are used to select 40 of the best genotypes for a given provenance.
Offspring from controlled crosses between trees in first generation orchards are tested in field trials and the best trees are then cloned and grafted onto root stock in advanced generation orchards. 

[ADD SITE AND SPU INFO]

```{r calculateorchardgenerations, echo=FALSE}
orchards_in_data <- read.csv("../../../phd/data/PhenologyAndPollenCounts/data_formatted_and_derived/derived_phenophase.csv", header=TRUE, stringsAsFactors = FALSE) %>% 
    select(Orchard, Year, ) %>%
    group_by(Orchard) %>%
    dplyr::summarise(Years = n_distinct(Year))
orchard_gen <- read.csv("../../../phd/data/OrchardInfo/OrchardGen.csv", header=TRUE, stringsAsFactors = FALSE)

orchard_meta <- dplyr::left_join(orchards_in_data, orchard_gen) %>%
    unique()

knitr::kable(orchard_meta, caption="Orchard generations reflecting the level of selection trees were subject to")

```

### selection only from a portion of the range
Provenances and sites included in my data exclude the coldest and wettest parts of the range. Figure \@ref(fig:siteprovclimate) shows the 1961-1990 climate normal mean annual temperature (MAT) and mean annual precipitation (MAP) for data from all lodgepole pine provenances in BC, provenances included in my data set, and sites where provenances were grown. Orchard sites are generally much warmer and drier than many locations where lodgepole pine grow. We may have difficulty extrapolating into the coolest and wettest parts of the lodgepole pine range, depending on actual yearly conditions at sites for which we have data. However, this may be advantageous when it comes to predicting phenology under climate change.

[SITE SELECTION AND CLIMATE CHANGE]

```{r siteprovclimate, fig.cap='(ref:siteprovclimate)', out.width='100%', echo=FALSE}
knitr::include_graphics('../../../phd/phd_proposal/figure/climate.png')
```

## no chilling
Previous models for lodgepole pine have fitted spring growth phenology models without considering chilling [@Chuine2001] and, in general, budburst models for boreal tree species have not required chilling to give accurate predictions [@Linkosalo2000].
For lodgepole pine growing in natural environments, chilling is always met and should continue to be met under the next century of climate change; 
in the AdapTree project, lodgepole pine seedlings grown for three to four weeks at 4 $^\circ\mathrm{C}$ met their chilling requirements.
Thus, I expect to be able to disregard the state of chilling and fit only Equations \ref{forcing} and \ref{simp_force_amt}).


## Naive climate change simulation

Climate change isn't evenly distributed across time and space. Use real climate change simulations and more sites.

## How can people use this model?
Use HPDI in this paper (easy). I'll publish the fstart and fend distributions in addition to the HPDI numbers (easy and good). I'll also publish the complete model input and output (flexible and pretty easy). People can also rerun the model (very flexible and probably hard).

## Discussion of potential applications

## Future directions
THE WHOLE RANGE

# Acknowledgements

Thanks to R. Johnstone for help with transcription. Workers with BC Seed Orchards for collecting such an extraordinary dataset and J. Woods, C. Walsh, R. Wagner, and J.Webber for assistance obtaining and interpreting the historical data. A. Elbakyan for background and resources. G. Hoare for technical support. T. Kerr for troubleshooting.


# References



